src/katsdpcalproc/pointing.py

```diff
@@ -236,23 +233,37 @@ def __call__(self, x):
         """
         return self._interp(x)
 
+# Helper function to calculate expected width
+
+
+def _get_widths(beamwidth, freqs, diameter):
+    expected_width = rad2deg(beamwidth * lightspeed
+                             / freqs / diameter)
+    expected_width = np.sqrt(2.0) * expected_width
+    # XXX This assumes we are still using default ant.beamwidth of 1.22
+    # and also handles larger effective dish diameter in H direction
```

Comment:
The XXX is to remind us to revisit this for MeerKAT+ with its differing dishes. Just so you know 🙂